app/services/manual/preview_service.rb

```diff
@@ -6,6 +6,11 @@ def initialize(user:, manual_id:, attributes:)
   end
 
   def call
+    manual = if manual_id
+               Manual.find(manual_id, user)
+             else
+               Manual.new(attributes)
+             end
```

Comment:
I don't think there's any reason it can't be a ternary expression. I'm going to leave it as it is but please feel free to change it as you see fit.